README.md

```diff
@@ -61,11 +61,18 @@ bundle exec guard
 ### Launch the program <a name = "launching"></a>
 From program directory:
 ```
-ruby lib/tictactoe.rb
+ruby tictactoe.sh
 ```
 
 ### Instructions
-- TBA
+
+Enter the number (1-9) of the space on the board where you want to move.
+
+Player 1 moves first and marks their spaces with an "X". Player 2 marks with an "O".
+
+To win, claim 3 adjacent spaces in a horizontal, vertical, or diagonal line.
+
+If there are no free spaces and no player has won, the game will end in a draw.
 
 <br>
```

Comment:
Is this block necessary?

Reply:
lol I didn't know if I should assume that the user had prior knowledge of tic tac toe but I'll remove it!

Reply:
I mean this `<br>`

Reply:
ah ok!